kratos/containers/container_expression/container_expression.h

```diff
@@ -140,7 +140,9 @@ class KRATOS_API(KRATOS_CORE) ContainerExpression {
     /**
      * @brief Move data from pBegin array to internal structure.
      *
-     * @warning This instance takes ownership of the passed array.
+     * @warning This instance does not take the ownership of the passed array. 
+     *  The life time of the passed array is not managed by this instance
+     * @throws Seg faults if this is used when passed @ref pBegin is destroyed. 
```

Comment:
segfaults aren't exceptions and won't get thrown so I'd highlight them as warnings instead.

Reply:
done :)